packages/@ama-sdk/schematics/schematics/typescript/shell/schema.json

```diff
@@ -28,6 +28,11 @@
       "description": "Skip NPM install",
       "default": false
     },
+    "skipGit": {
```

Comment:
Thank you @matthieu-crouzet for your comment! I have added the `commit` option in the shell generator schematics. However, it is currently not used anywhere.

I have some questions:
- Should the `commit` option of the shell generator schematics be `true` by default?
- Should the `npm create @ama-sdk` command create a commit?
- Should the `npm create @ama-sdk` command use the `commit` option of the schematics to create a commit?
- If the answer to the previous question is yes, the commit would only contain the shell files and neither the result of `yarn set version` nor the output of sdk generation... so for those extra changes, we could either create another commit or amend the previous one?

I personally prefer not creating a commit in `npm create` (as it is done in the current state of this PR), but if it is decided to create one, then I think it would be better to create it directly with the right files (calling the shell generator schematics with the `commit` option set to `false`) and creating the commit at the end, rather than amending a commit. Note that if a repository already exists in a parent folder, the `RepositoryInitializerTask` does nothing, so we should be especially careful not to amend a previous commit in this case.

Reply:
Indeed it should not be part of the `shell` schematic but more of the `create` script at the end with all the files.
It's the behavior that we have by default when run `npm create @o3r` if I'm not mistaken.

Reply:
`npm create @o3r` creates a commit, but does not include all changes in it.

Reply:
In your opinion should we create an issue to handle it later
I think it could be great if the repo is "clean" when you finish the creation